docs/src/components/Markdown.js

```diff
@@ -6,8 +6,11 @@ import highlightjs from 'highlight.js';
 import './Markdown.css';
 import 'highlight.js/styles/a11y-light.css';
 
+const sidebarIndex = require('./sidebarIndex.js');
```

Comment:
@ayeshakmaz nit - can we use a regular `import` in this case? In `scripts/` we do have to use `require` since those don't go through webpack

Reply:
oh whoops sure thing